scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -152,16 +152,29 @@ function doAddPlugin() {
   fi
 }
 
+# $1 no-backgrounding flag -nb
 function doStartEclipse() {
-  doConfigureEclipse -u
-  echo "launching Eclipse in workspace ${WORKSPACE} at ${WORKSPACE_PATH}"
-  if doIsWindows
+  if [ "${1}" = "-nb" ]
```

Comment:
what option is `-nb` for? You did not add it to the help usage.

Reply:
Please also stay conform with POSIX conventions:
https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

Reply:
It launches the Eclipse instance without `start` (Windows) or an appended `&` (macOS/Linux).
So the terminal waits until the Eclipse instance is terminated (required for automated file cleanup).

Do we add deatails to internal functions to the documentation usually?
The function `doConfigureEclipse` also allows for an optional `-u` flag, which isn't documented anywhere (or maybe I've overlooked it) ...